cilium/ebpf.io-summit
# feat: add the Host a watch party page

**Describe what changes this pull request brings**
This pull request adds the `Host a watch party` page.

**Steps to test**
1. Open the [page](https://deploy-preview-35--ebpf-summit-2021.netlify.app/host-a-watch-party/)
2. Confirm that everything looks and works as expected

**Screenshots**
<img width="1600" alt="EBPF-summit-home-1600@1x" src="https://user-images.githubusercontent.com/17677196/181782817-b520fdac-aaab-4e9a-b1e0-9fd526f37af6.png">
